incomplete_cooperative/tests/test_generators.py

```diff
@@ -14,12 +17,24 @@
 from incomplete_cooperative.normalize import normalize_game
 from incomplete_cooperative.protocols import Game, Value
 
+EPSILON = 1e-10
+
 
 class GeneratorsTests:
     generator: Callable[[int], Callable[[int, Generator], Game]]
     is_random: bool = True
     implements_generator: bool = True
 
+    def test_superadditive(self):
```

Comment:
Proc toto?

Reply:
Pro jistotu, Testuju jestli to co vygeneruju je opravdu superadditivní (vracím jakoby mínus xos funkce, takže jsou superadditivní)

Reply:
Ta funkce na testování SA tam možná měla být už dlouho, ale teď obzvlášť jak vlastně přecházíme mezi sub a super additivníma to tam chci mít takhle explicitně všechno otestovaný, že někde neděláme blbost